tests/knr/2d_arrays.d.ref

```diff
@@ -0,0 +1,37 @@
+unittest {
```

Comment:
unittests are functions, so the brace should be on a separate line

Reply:
The idea with having function signatures on a separate line is to make them stand out while visually scanning the file. This doesn't really apply to unit test "functions".